src/cpp/fastdds/builtin/type_lookup_service/TypeLookupReplyListener.hpp

```diff
@@ -17,11 +17,13 @@
  *
  */
 
-#ifndef TYPELOOKUP_REPLY_LISTENER_HPP_
-#define TYPELOOKUP_REPLY_LISTENER_HPP_
-#ifndef DOXYGEN_SHOULD_SKIP_THIS_PUBLIC
+#ifndef _FASTDDS_TYPELOOKUP_SERVICE_REPLY_LISTENER_HPP_
```

Comment:
The criteria is to include the path and file name. Please fix it in the other files too.

```suggestion
#ifndef _FASTDDS_BUILTIN_TYPE_LOOKUP_SERVICE_TYPE_LOOKUP_REPLY_LISTENER_HPP_
```

Reply:
fixed in [29681bc](https://github.com/eProsima/Fast-DDS/pull/4240/commits/29681bc3ecb5a0cf1f4e298b7bf093bff1ab1753)